_std/defines/gang.dm

```diff
@@ -0,0 +1,161 @@
+// If they see a player, they will remember them until the next GANG_TAG_SCORE_INTERVAL.
+// Once the next GANG_TAG_SCORE_INTERVAL has elapsed, all memorised players provide heat, then are forgotten.
+
+/// how often gang tags search for nearby people, in deciseconds
+#define GANG_TAG_SCAN_RATE 10 DECI SECONDS
+/// how often tags calculate their heat & score, in seconds
+#define GANG_TAG_SCORE_INTERVAL 15 SECONDS
+/// How many points a tag gives for each heat rating it has
+#define GANG_TAG_POINTS_PER_HEAT 1.5
+/// How much heat gang tags retain every score interval
+// Higher means gang tags stay hot for longer
+// If popular gang tags are staying too hot for too long after players leave, consider setting it lower.
+#define GANG_TAG_HEAT_DECAY_MUL 0.9
+
+// MATH FOR NERDS:
+// https:// www.desmos.com/calculator/p9uv6debrp
+// The original math for heat level is as follows, (located in the gangtag's 'apply_score' in gangwar.dm):
+// TAG HEAT LEVEL = log(10,10*X)*5
+// Where X is the % of how hot said tag is, compared to the hottest tag
+// This roughly translates to:
+// Having the hottest tag gets level '5'
+// Having 63% of the top heat is level '4'
+// Having 40% of the top heat is level '3'
+// Having 25% of the top heat is level '2'
+// Having 10% of the top heat is level '1'
+// Having less than 10% is level '0'
+
+
+// WHAT THIS MEANS:
+// Every gang tag can provide anywhere between 0-6 * GANG_TAG_POINTS_PER_HEAT per GANG_TAG_SCORE_INTERVAL.
+// You can therefore estimate how many points a gang might have using this, A VERY successful gang may see mostly level 4 tags
+
+// Giving a gang 2 more spray bottles will therefore mean 2 more level 3~4 tags - 3.5*GANG_TAG_POINTS_PER_HEAT points every GANG_TAG_SCORE_INTERVAL
+// With the default settings (scanrate 10, interval 15, points per heat 3), gangs will get roughly
+// Use this to figure if your item is too expensive!
+
+
+
+// GANG TAG SIZES:
+
+
+
+
+/// Radius of the circle that tags claim
+#define GANG_TAG_INFLUENCE_LOCKER 4
+/// Radius of the circle that gang tags can see inside (can't be sprayed inside)
+#define GANG_TAG_SIGHT_RANGE_LOCKER 0
+// keep in mind, smaller maps will still have fewer players & less gangs
+
+// overriding gang tag sizes, 15-8 seems fair for most highpop maps
+#ifdef MAP_OVERRIDE_COGMAP2
+	#define GANG_TAG_INFLUENCE 15
+	#define GANG_TAG_SIGHT_RANGE 8
+
+#elif defined(MAP_OVERRIDE_DONUT3)
+	#define GANG_TAG_INFLUENCE 15
+	#define GANG_TAG_SIGHT_RANGE 8
+#elif defined(MAP_OVERRIDE_OZYMANDIAS) //jesus christ. this is HUGE.
+	#define GANG_TAG_INFLUENCE 20
+	#define GANG_TAG_SIGHT_RANGE 12
+#elif defined(MAP_OVERRIDE_ATLAS)
+	#define GANG_TAG_INFLUENCE 10
+	#define GANG_TAG_SIGHT_RANGE 5
+#else
+	#define GANG_TAG_INFLUENCE 12
+	#define GANG_TAG_SIGHT_RANGE 6
+#endif
+
+// shorthands for range calcs
+#define GANG_TAG_INFLUENCE_SQUARED GANG_TAG_INFLUENCE*GANG_TAG_INFLUENCE
+#define GANG_TAG_SIGHT_RANGE_SQUARED GANG_TAG_SIGHT_RANGE*GANG_TAG_SIGHT_RANGE
```

Comment:
Unless these are for some reason needed across multiple files they'd be better as local defines where they're used.